<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonManageClass</class>
 <widget class="QMainWindow" name="PersonManageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简单人事管理</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5">
     <widget class="QCheckBox" name="isSearch">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="search">
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="searchCondition"/>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="6">
     <widget class="QTableView" name="tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addpersonnel"/>
   <addaction name="deletePersonnel"/>
   <addaction name="editPersonnel"/>
   <addaction name="view"/>
   <addaction name="Undo"/>
   <addaction name="Redo"/>
  </widget>
  <action name="addpersonnel">
   <property name="icon">
    <iconset theme="contact-new"/>
   </property>
   <property name="text">
    <string>添加个人信息</string>
   </property>
   <property name="toolTip">
    <string>添加个人信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="deletePersonnel">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>删除个人信息</string>
   </property>
   <property name="toolTip">
    <string>删除个人信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="editPersonnel">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>修改人员信息</string>
   </property>
   <property name="toolTip">
    <string>修改人员信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="Undo">
   <property name="icon">
    <iconset theme="document-revert"/>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="Redo">
   <property name="icon">
    <iconset theme="edit-redo"/>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
   <property name="toolTip">
    <string>还原</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="view">
   <property name="icon">
    <iconset theme="dialog-error"/>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
   <property name="toolTip">
    <string>查看</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PersonManage.qrc"/>
 </resources>
 <connections/>
</ui>
